src/acquisition/covidcast_nowcast/load_sensors.py

```diff
@@ -33,17 +36,20 @@ def main(csv_path: str = SENSOR_CSV_PATH) -> None:
     """
     user, pw = secrets.db.epi
     engine = sqlalchemy.create_engine(f"mysql+pymysql://{user}:{pw}@{secrets.db.host}/{DB_NAME}")
-    for filepath, attributes in CsvImporter.find_csv_files(csv_path):
-        if attributes is None:
-            move(filepath, filepath.replace("receiving", "archive/failed"))
+    for filepath, attr in CsvImporter.find_csv_files(csv_path):
```

Comment:
does this rename have a semantic purpose, or just to save typing?

Reply:
i had shortened it to be lazy about a line that was too long, but fixed it properly. ill change it back.